include/seqan3/core.hpp

```diff
@@ -31,15 +31,28 @@
 // DAMAGE.
 //
 // ============================================================================
-// Author: Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+// Author: Rene Rahn <rene.rahn AT fu-berlin.de>
```

Comment:
:middle_finger: 
No, seriously, please remove the line according to our new guidlines. instead add author to the doxygen.